src/breadcrumb-group/item/item.tsx

```diff
@@ -28,40 +27,29 @@ const BreadcrumbItemWithPopover = <T extends BreadcrumbGroupProps.Item>({
   itemAttributes,
   children,
 }: BreadcrumbItemWithPopoverProps<T>) => {
-  const [showPopover, setShowPopover] = useState(false);
-  const textRef = useRef<HTMLElement>(null);
-  const popoverContent = (
-    <Tooltip trackRef={textRef} value={item.text} size="medium" onDismiss={() => setShowPopover(false)} />
-  );
-
-  useEffect(() => {
-    if (showPopover) {
-      return registerTooltip(() => {
```

Comment:
Do we need to remove this registry logic, and in the scope of this PR? _I think_ the purpose of this was to make sure two tooltips are never seen at the same time. Is there no chance we are not introducing regressions?

Reply:
Or is this just handling hiding the tooltip on pressing Escape which is already handled by the tooltip [here](https://github.com/cloudscape-design/components/blob/0e10faf3e2a2463ebbacf77a09eabb3de5003781/src/internal/components/tooltip/index.tsx#L41-L63)?